### PR TITLE
Add initial SECURITY.md file

### PR DESCRIPTION
**Pull request by vpereira**
To better describe our security issues handling, use a standard
way to define it via SECURITY.md

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  * [Comments on this Policy](#comments-on-this-policy)
+
+## Reporting a Bug
+
+The `standard` team and community take all security bugs in `Open Build Service` seriously.
+Thank you for improving the security of `Open Build Service`. We appreciate your efforts and
+responsible disclosure and will make every effort to acknowledge your
+contributions.
+
+Report security bugs by emailing the [Security Team](https://www.suse.com/de-de/support/security/contact/) at security@suse.com.
```
Comment (andreasstieger) [inline SECURITY.md]: Remove "/de-de" please

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+This document outlines security procedures and general policies for the `standard`
+project.
+
+  * [Reporting a Bug](#reporting-a-bug)
+  * [Disclosure Policy](#disclosure-policy)
+  * [Comments on this Policy](#comments-on-this-policy)
+
+## Reporting a Bug
+
+The `standard` team and community take all security bugs in `Open Build Service` seriously.
```
Comment (hennevogel) [inline SECURITY.md]: What's the `standard` team? :-)

File: SECURITY.md (hunk 1 above)
Comment (hennevogel) [inline SECURITY.md]: I think we need to make a bite more clear that this is a 'service' that **SUSE** provides for us. Our connection with the company might not be so clear for everybody.

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Report security bugs by emailing the [Security Team](https://www.suse.com/de-de/support/security/contact/) at security@suse.com.
+
+The security team will acknowledge your email as soon as possible, and will send a
+more detailed response indicating the next steps in handling
+your report. After the initial reply to your report, the security team will
+endeavor to keep you informed of the progress towards a fix and full
+announcement, and may ask for additional information or guidance.
+
+Report security bugs in third-party modules to the person or team maintaining
```
Comment (hennevogel) [inline SECURITY.md]: What are third-party modules? I guess this would warrant some examples..

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  * Confirm the problem and determine the affected versions.
+  * Audit code to find any potential similar problems.
+  * Prepare fixes for all releases still under maintenance. These fixes will be
+    deployed in production and released as fast as possible to all maintained
+    versions of Open Build Service.
+
+## Comments on this Policy
+
+If you have suggestions on how this process could be improved please submit a
+pull request.
```
Comment (hennevogel) [inline SECURITY.md]: ...or discuss it with us on our mailinglist [obs-devel@opensuse.org](https://lists.opensuse.org/obs-devel/).

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
+# Security Policies and Procedures
+
+This document outlines security procedures and general policies for the `standard`
```
Comment (dmarcoux) [inline SECURITY.md]: By the `standard` project, do we mean our `OBS` instance? I'm not so sure I understand this.

File: SECURITY.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+When the security team receives a security bug report, they will assign it to a
+primary handler. This person will coordinate the fix and release process,
+involving the following steps:
+
+  * Confirm the problem and determine the affected versions.
+  * Audit code to find any potential similar problems.
+  * Prepare fixes for all releases still under maintenance. These fixes will be
+    deployed in production and released as fast as possible to all maintained
+    versions of Open Build Service.
```
Comment (dmarcoux) [inline SECURITY.md]: `Open Build Service` should be quoted again here to be constant throughout the document.

File: SECURITY.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+## Comments on this Policy
+
+If you have suggestions on how this process could be improved please submit a
```
Comment (dmarcoux) [inline SECURITY.md]: Missing comma between 'improved' and 'please'

File: SECURITY.md (hunk 3 above)
Comment (hennevogel) [reply SECURITY.md]: I know that. People don't, that's why we should include examples. Like:

Report security bugs in third-party modules (perl modules, ruby gems, used libraries) to the person or team maintaining them (upstream), not use please.

**Comment by adrianschroeter** [comment]
On Montag, 27. August 2018, 14:34:46 CEST wrote Henne Vogelsang:
> hennevogel commented on this pull request.
> 
> 
> 
> > +## Reporting a Bug
> +
> +The `standard` team and community take all security bugs in `Open Build Service` seriously.
> +Thank you for improving the security of `Open Build Service`. We appreciate your efforts and
> +responsible disclosure and will make every effort to acknowledge your
> +contributions.
> +
> +Report security bugs by emailing the [Security Team](https://www.suse.com/de-de/support/security/contact/) at security@suse.com.
> +
> +The security team will acknowledge your email as soon as possible, and will send a
> +more detailed response indicating the next steps in handling
> +your report. After the initial reply to your report, the security team will
> +endeavor to keep you informed of the progress towards a fix and full
> +announcement, and may ask for additional information or guidance.
> +
> +Report security bugs in third-party modules to the person or team maintaining
> 
> I know that. People don't, that's why we should include examples. Like:
> 
> Report security bugs in third-party modules (perl modules, ruby gems, used libraries) to the person or team maintaining them (upstream), not use please.

Well, it doesn't matter so much here since security@suse.com will needs handle
it anyway. So, I would just drop the sentence.

(And we are also responsible in case the package maintainer does not react and we 
 need to fix/update our package copy anyway.)

-- 

Adrian Schroeter
email: adrian@suse.de

SUSE Linux GmbH, GF: Felix Imendörffer, Jane Smithard, Graham Norton, HRB 21284 (AG Nürnberg) 
 
Maxfeldstraße 5                         
90409 Nürnberg 
Germany

File: SECURITY.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+  * [Comments on this Policy](#comments-on-this-policy)
+
+## Reporting a Bug
+
+The `Open Build Service` team and community take all security bugs in `Open Build Service` seriously.
+Thank you for improving the security of `Open Build Service`. We appreciate your efforts and
+responsible disclosure and will make every effort to acknowledge your
+contributions.
+
+Report security bugs by emailing the [Security Team](https://www.suse.com/support/security/contact/) at security@suse.com.
```
Comment (hennevogel) [inline SECURITY.md]: How about: 

The OBS as openSUSE project makes use of the [SUSE/openSUSE security team](https://www.suse.com/support/security/contact/). So please report security problems by contacting them.

File: SECURITY.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+  * Confirm the problem and determine the affected versions.
+  * Audit code to find any potential similar problems.
+  * Prepare fixes for all releases still under maintenance. These fixes will be
+    deployed in production and released as fast as possible to all maintained
+    versions of `Open Build Service`.
+
+## Comments on this Policy
+
+If you have suggestions on how this process could be improved, please submit a
+pull request.
```
Comment (hennevogel) [inline SECURITY.md]: I really don't want to have discussions about policies in PRs. Please change this to the obs-devel mailinglist.

File: SECURITY.md (hunk 8 above)
Comment (saraycp) [reply SECURITY.md]: I agree, steps to follow are set by SUSE. Their instructions or contact email could change and we wouldn't have to change our document.

Comment (vpereira) [reply SECURITY.md]: after working 5 years in the security team, IMO, it is easier the contact website to change/go offline than the e-mail address ```security@suse.com```

Comment (saraycp) [reply SECURITY.md]: Sorry, I deleted my previous comment accidentally. It was:

> I agree, steps to follow are set by SUSE. Their instructions or contact email could change and we wouldn't have to change our document.

But @vpereira, yours is a good reason too :thinking:

Comment (vpereira) [reply SECURITY.md]: @hennevogel  are we good to go here?

Comment (hennevogel) [reply SECURITY.md]: If you tell me the encrypting the mail does not matter so much, sure :-)

**Pull request merged**